docs/photoniq/prerendering/monitor-prerendering/view-prefetching.md

```diff
@@ -0,0 +1,92 @@
+
+## View Prefetching in Progress
+
+<Tabs groupId="operating-systems">
+<TabItem value="console" label="Web Console">
+
+Follow these instructions to view render history in the web console.
+
+1. Log in to your Dynamic Prerendering dashboard.
+2. Click **Prefetching**.
+3. Select the origin that you want to view.
+4. If necessary, click **In Progress**.
+
+![Prerendering Prefetching In Progress Tab](/img/prerendering/prefetching-in-progress.png)
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to:
+
+- [Get prefetch statistics](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-metrics-stats-prefetch-origin/get)
+- [Get prefetch URLs](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-origins-origin--prefetch-urls/get)
+
+</TabItem>
+</Tabs>
+
+## In Progress Fields
+
+Dynamic Prerendering provides the following prefetching fields. Field names in the web console are different than those in the API responses. Refer to the API endpoint documentation for a full explanation of the API response.
+
+### Stats
+
+- **Total** - Total number of URLs to be prefetched.
+- **Completed** - Number of URLs successfully prefetched.
+- **Queued** - Number of URLs queued for prefetch.
+- **Failed** - Number of URLs where prefetching failed.
+- **Retrying** - Number of URLs retrying prefetch.
+
+### Events
+
+- **Timestamp** - When the prefetch URL was created.
+- **Status** - URL status.
+- **URL** - Origin URL.
+- **Device** - Icon indicating whether the prefetch was rendered for desktop or mobile.
+
+## View Prefetching History
+
+<Tabs groupId="operating-systems2">
+<TabItem value="console" label="Web Console">
+
+Follow these instructions to view prefetch history in the web console.
+
+1. Log in to your Dynamic Prerendering dashboard.
+2. Click **Prefetching**.
+3. Select the origin that you want to view.
+4. If necessary, click **History**.
+
+![Prerendering Prefetching History Tab](/img/prerendering/prefetching-history.png)
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to:
+
+- Get prefetch statistics
+- Get prefetch URLs
+
+</TabItem>
+</Tabs>
+
+## Prefetching History Fields
+
+Dynamic Prerendering provides the following prefetching history fields. Field names in the web console are different than those in the API responses. Refer to the API endpoint documentation for a full explanation of the API response.
+
+- Type - Whether prefetching was scheduled or done manually.
+- **Total** - Total number of URLs to be prefetched.
+- **Completed** - Number of URLs successfully prefetched.
+- **Queued** - Number of URLs queued for prefetch.
+- **Failed** - Number of URLs where prefetching failed.
+- **Status** - Success status (completed, in progress, failed).
+- **Created At** - When the prefetch URL was created.
```

Comment:
This page displays the list of prefetching processes and their results. Here, we summarize all processes, grouping the URLs by status. That said, it does not make sense to have the `Created At` mentioning `prefetch URL`, since this is not representing a single URL.
I have a few suggestions:
```suggestion
- **Created At** - When the prefetch process was initiated/started/triggered (one of these 3 action)
- **Created At** - When the prefetch was initiated/started/triggered (one of these 3 action)
```